Reason: Fix upstream issue 334, missing function declaration in TrayWindow.h
Upstream: Yes, apllied in r509.
Source: qiaomuf
diff -urN fcitx-4.0.0/src/ui/TrayWindow.h fcitx-4.0.0-patched/src/ui/TrayWindow.h
--- fcitx-4.0.0/src/ui/TrayWindow.h	2010-10-11 14:43:19.000000000 +0800
+++ fcitx-4.0.0-patched/src/ui/TrayWindow.h	2010-11-19 22:07:39.445075018 +0800
@@ -54,6 +54,7 @@
 void DeInitTrayWindow(TrayWindow *f_tray);
 void RedrawTrayWindow(void);
 void TrayEventHandler(XEvent* event);
+void DestroyTrayWindow();
 
 extern TrayWindow tray;
 
